packages/tokens/src/fungible/storage.rs

```diff
@@ -340,15 +342,15 @@ impl Base {
     /// # Events
     ///
     /// * topics - `["transfer", from: Address, to: Address]`
-    /// * data - `[amount: i128]`
+    /// * data - `{ to_muxed_id: Option<u64>, amount: i128 }: Map`
```

Comment:
same here, we should get rid of `Map` notation here (at least for the scope of this PR)